src/RazorSdk/Targets/Microsoft.NET.Sdk.Razor.StaticWebAssets.targets

```diff
@@ -536,7 +536,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <MSBuild Condition="$(_HasProjectsWithStaticWebAssetPublishTargets)"
       Targets="%(_ProjectConfigurationsWithPublishTargets.GetPublishAssetsTargets)"
       Properties="%(_ProjectConfigurationsWithPublishTargets.AdditionalPublishProperties)"
-      RemoveProperties="%(_ProjectConfigurationsWithPublishTargets.AdditionalPublishPropertiesToRemove)"
+      RemoveProperties="%(_ProjectConfigurationsWithPublishTargets.AdditionalPublishPropertiesToRemove);$(_GlobalPropertiesToRemoveFromProjectReferences)"
```

Comment:
We might also need to do this in the pack target files (for completeness), but do not do it if you do not want it, as I do not think there is a current scenario where it matters.

Reply:
I think I'll leave it as is, as testing this scenario is a bit convoluted. Tried a bunch of things to repro this (my machine, cti machine, cti app, cti publish profile, etc.) without any luck. Ended up having CTI run the dogfood sdk on their local env to validate this fix.